app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,69 @@
+class CustomersController < ApplicationController
+  # protect_from_forgery with: :null_session
+
+  def index
+    customers = Customer.all
+
+    render(
+      # json: customers.as_json(except: [:created_at, :updated_at]),
+      json: customers,
+      status: :ok
+    )
+  end
+
+  def show
+    customer = Customer.find_by(id: params[:id])
+    if customer
+      render(
+        json: customer,
+        # json: customer.as_json(except: [:created_at, :updated_at]),
+        status: :ok
+      )
+    else
+      render(
+        json: {nothing: true}, status: :not_found
+      )
```

Comment:
This will literally send back `{ nothing: true }` as a JSON response. Not sure that's what you intended.